Reports/progress_report.md

```diff
@@ -0,0 +1,296 @@
+
+#### Goal
+- Supporting the user community of the DSH codebase
+- Publicising our work via blogposts, reports or papers
+- Communicating our work through conference/workshop talks or posters 
+
+#### Progress
+This story has changed in scope along the year, work done here has been tha tof presenting DSH externally via events and talks.
+
+Yet we have identified work to be done within this story to define and establish a DSH community and user base, what this means and entails needs to be discussed and agreed yet.
+
+- We have established a shared slack channel with UCL to discuss common approaches to information governance processes
+-  [AI UK demonstration proposal](https://thealanturininstitute.sharepoint.com/:x:/s/SafeHaven/EfKD3w8Gi9NFv6JBshOkugsBOnn4v3ZdU-FTeIcy5obQcg?e=Rhq9w4)
+  - [proposal collaborative note](https://hackmd.io/AmcYdsyETU2dVgtIdfVL-g)
+  - We are following a similar format to last year but want to bring forward the community work and the satre specification. We want to have an interactive activity that blends role playing the different stakeholder groups and collectively deciding on specification features. The demo challenge last year did not work so the technical side this year will be demonstrated by a video and project members "touring" the repositories, docs and environment
+- Met with Nottingham to support them as users of the DSH codebase
+- The team worked together on the content for DSH activities on RSEcon  as well as the UK TRE community satellite event #46 . 
+- The team visited the Bennett Institute for a show and tell about DSH and OpenSafely https://github.com/alan-turing-institute/tps-project-management/issues/157
+  - No immediate collaboration but agreed to be involved in the specification and TRE community
+- For discussion: this story needs to be redefined to include external engagement or a new one created
+
+## TRESA
+Over the year TRESA have increased its autonomy from the DSH research project, in terms of work ownership and management.
+Therefore TRESA stories have not been indepently updated and it is more comprenhensive to update on the service area as a whole.
+
+This warrants updating and reviewing the stories we keep under the DSH roadmap, focusing on communicating with the service area rahter than planning or prioritising for them.
+
+### Updates
+A new team of have been onboarded, who have focused on learning the ropes while simultaneously supporting the December DSG.
+With some support from the DSH team they were able to do so.
+
+We had a detailed[ handover meeting](https://hackmd.io/kh6siuZcTdCxcfYryAvypw) to go over each process, the idea is that moving forward TRESA directly owns processes and their improvement.
+
+We have also identified areas of work for TRESA to fully establish itself as a service area, these need to be elaborated into a full proposal with sprints and stories but main areas are:
```

Comment:
Very cool!

Reply:
I feel I need to catch up with @helendduncan and @jemrobinson about this.

I sketched out some "sprints" for TRESA to try and start moving from being purely reactionary to making progress. I want to avoid stepping on @helendduncan's toes though and don't want to be dictating what TRESA's focus is if that isn't my role here.

Reply:
@KirstieJane would you like/be able to be involved in planning for TRESA like you have for DSH?

Reply:
Happy to be involved if I can be useful! Feel free to ping me on slack if you're looking to find a time to meet 💖